spec/features/sign_in_spec.rb

```diff
@@ -32,20 +32,21 @@
     end
 
     after(:each) do
-      expect(page).to have_current_path(new_user_session_path)
-      expect(page).to have_content( 'Invalid Email or password' )
+      expect(page).to have_current_path(root_path)
```

Comment:
This should be an scenario by itself. Always try to have one expect per scenario.